### PR TITLE
feat: Add screenshare filmstrip

### PR DESCRIPTION
**Pull request by robertpin**
Add new screen share layout with resizable top panel
Only enable new layout in large meetings (min 50 participants - configurable)

### CODE REVIEW

**Comment by robertpin** [comment]
Jenkins test this please

File: react/features/conference/components/web/ToggleTopPanelLabel.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+import { IconMenuDown } from '../../../base/icons';
+// @ts-ignore
+import { Label } from '../../../base/label';
+// @ts-ignore
+import { Tooltip } from '../../../base/tooltip';
+// @ts-ignore
+import { setTopPanelVisible } from '../../../filmstrip/actions.web';
+
+const ToggleTopPanelLabel = () => {
+    const asd = 'hello';
```
Comment (quitrk) [inline react/features/conference/components/web/ToggleTopPanelLabel.tsx:15]: whoops?

File: react/features/conference/components/web/ToggleTopPanelLabel.tsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    const dispatch = useDispatch();
+    const { t } = useTranslation();
+    const topPanelVisible = useSelector((state: any) => state['features/filmstrip'].topPanelVisible);
+    const onClick = useCallback(() => {
+        dispatch(setTopPanelVisible(true));
+    }, []);
+
+    return !topPanelVisible && (<Tooltip
```
Comment (quitrk) [inline react/features/conference/components/web/ToggleTopPanelLabel.tsx:23]: kinda weird to use it like this, maybe rename to topPanelHidden and negate the selector result?

File: react/features/conference/components/web/ToggleTopPanelLabel.tsx (hunk 1 above)
Comment (quitrk) [inline react/features/conference/components/web/ToggleTopPanelLabel.tsx:15]: leftover here

**Review by quitrk — APPROVED** [review]

**Pull request merged**